Mathlib/FieldTheory/Galois/Basic.lean

```diff
@@ -539,7 +540,8 @@ instance (F K : Type*) [Field F] [Field K] [h : IsQuadraticExtension F K] :
 /--
 A quadratic extension has abelian Galois group.
 -/
-instance (F K : Type*) [Field F] [Field K] [IsQuadraticExtension F K] :
+instance IsQuadraticAlgebra.isMulCommutative_galoisGroup (F K : Type*) [Field F] [Field K]
```

Comment:
Can you add the isomorphism with.. *the* group of order two?

Reply:
Or maybe it is ` Multiplicative (ZMod 2)`?

Reply:
I guess the latter.

Reply:
But is this really needed now if we cannot even agree on what is needed?

Reply:
If we can't agree then I can add an isomorphism with *any* group of order 2 😆

Reply:
IMHO We can leave it to the first person who'll need this. At worst, the second person will fix it, and then we'll be done.

Reply:
Yes sorry, I didn't realize you proved it has order 2. Let's forget about this.